### DISCUSSION TITLE
How to verify my student status as a distance learning program learner

### DISCUSSION THREAD

**Discussion by charles2648** [community/community#58970] [GitHub Education]
### Hi everyone,

Hello,

I am a Gatech OMSCS student who wanna apply for GitHub Education student pack, however, I got rejected for the reasons shown in the image. I uploaded school issued enrollment certificate for verification, but the certificate does not show that my program is a distance learning program. 

I tried using unofficial transcript but the transcript does not show my program end date. How can I verify my status?

Thank you
<img width="490" alt="image" src="https://github.com/community/community/assets/17574313/c3c64de1-8dd8-4ac2-a186-fbc233fe25b0">

**Comment by ren-paz** [comment]
Same issue here, not sure what's going on with GitHub right now but they're clearly redesigning everything. Despite signing up for the student pack last month before they set up this awful system, I have to nevertheless reverify so that I can renew my github pro coupon. Like you, I have no way of showing that I am a virtual program student with their incredibly specific requirements (name of school plus date on transcript, webcam only???).

The only fix I can think of is applying for a buzzport student ID, however GT is not offering those until fall of this year. So if you're in a similar situation as me and your github pro coupon expires this month then you're SOL.

**Comment by samus-aran** [comment]
Hi @charles2648 👋 

You are able to submit multiple pieces of evidence. If possible take a photos of both pieces beside one another and submit them as the most suitable evidence type.

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

Alternatively you can reach out to your institution to provide evidence that your course is online and remote learners attend it.

**Reply by hardianlawi** [reply]
<img width="605" alt="image" src="https://github.com/community/community/assets/22806872/8ee7ad2a-73f3-4d93-ae21-df8bae36d4c9">

There's no option for me to select proof type at all. Is this a bug?

Also, why is there no more explanation about the school and location detected? I'm currently enrolled in an online master programme so it's hard to tell what's wrong based on the information given since I'm sure the documents I submitted would be approved if the verifiers are actual humans. :')
